addons/project_todo/static/tests/todo_conversion_form_view.test.js

```diff
@@ -11,7 +11,7 @@ defineTodoModels();
 
 beforeEach(() => {
     ProjectTask._views = {
-        form: `
+        "form,1": `
```

Comment:
```suggestion
        "form, 1": `
```

Reply:
By applying space, it cannot find the view because it considers the space also. So, it does not need to change.